scripts/reactor_deployment.py

```diff
@@ -219,12 +266,19 @@ def pre_det_deployment(df, base_col, ar_dict, greedy=True):
     -------
     df: pandas dataframe
         The dataframe of capacity information with the deployed reactors.
+
+    Notes
+    -----
+    * This function assumes that there is a column called 'Year' that is not
+    the column of indices.
     """
     # initialize the number of reactor columns
     df = reactor_columns(df, ar_dict)
 
+    start_index = df[df['Year'] == dep_start_year].index[0]
```

Comment:
Is this implementation completely different from other versions in this file? It looks like a repeated pattern which could be succinctly written in its own function, tested independently, and then called here. Just food for thought.

Reply:
We love smaller functions!